### PR TITLE
Probably nerfs DNA vault stacking and speed ability

### PR DESCRIPTION
**Pull request by SomeguyManperson**
:cl:  
tweak: DNA vaults can't be stacked, one use per body
tweak: DNA vault speed upgrade is slightly slower
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/human_defines.dm [hunk 1]
```diff
@@ -47,6 +47,8 @@
 
 	var/datum/physiology/physiology
 
+	var/can_use_vault = 1
```
Comment (Bibby0110) [inline code/modules/mob/living/carbon/human/human_defines.dm]: var/can_use_vault = TRUE

File: code/modules/station_goals/dna_vault.dm [hunk 2]
```diff
@@ -205,11 +205,14 @@
...
 	data["choiceB"] = ""
 	if(user && completed)
-		var/list/L = power_lottery[user]
-		if(L && L.len)
-			data["used"] = FALSE
-			data["choiceA"] = L[1]
-			data["choiceB"] = L[2]
+		if(ishuman(user))
+			var/mob/living/carbon/human/H = user
+			if(H.can_use_vault == TRUE)
```
Comment (Bibby0110) [inline code/modules/station_goals/dna_vault.dm]: no need for the == TRUE

**Comment by Bibby0110** [comment]
I'm sorry but I had to do it

**Comment by Bibby0110** [comment]
sexy

**Comment by Ratvar** [comment]
I know the true and only nerf to speed boosts: replacing it with something that's not

File: code/modules/mob/living/carbon/human/human_defines.dm [hunk 3]
```diff
@@ -47,6 +47,8 @@
 
 	var/datum/physiology/physiology
 
+	var/can_use_vault = TRUE
```
Comment (Amelia0010) [inline code/modules/mob/living/carbon/human/human_defines.dm]: just have a static list on the DNA vault of all users instead, and only allow people to use it if they're not in said list

**Review by Amelia0010 — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/human/human_defines.dm (hunk 3 above)
Comment (alexkar598) [reply code/modules/mob/living/carbon/human/human_defines.dm]: Didn't we fase out static vars for GLOBs :)

Comment (Amelia0010) [reply code/modules/mob/living/carbon/human/human_defines.dm]: har har har har har har very funny

**Review by Amelia0010 — APPROVED** [review]

**Pull request merged**